src/components/GisidaLite/index.tsx

```diff
@@ -73,10 +73,6 @@ const GisidaLite = (props: GisidaLiteProps) => {
     onMouseMoveHandler,
   } = props;
 
-  if (mapCenter === undefined) {
-    return <Loading />;
-  }
-
```

Comment:
What caused the change in this file?

Reply:
moved hooks above this code. Its an erly return and thus makes it possible for some hooks not to be called in the same order during certain executions